great_expectations_cloud/agent/models.py

```diff
@@ -47,6 +51,13 @@ class RunCheckpointEvent(EventBase):
     splitter_options: Optional[Dict[str, Any]] = None
 
 
+class RunScheduledCheckpointEvent(ScheduledEventBase):
+    type: Literal["run_scheduled_checkpoint.received"] = "run_scheduled_checkpoint.received"
+    datasource_names_to_asset_names: Dict[str, Set[str]]
+    checkpoint_id: uuid.UUID
+    splitter_options: Optional[Dict[str, Any]] = None
+
```

Comment:
Is there a correlation id in this as well of is that only present in the message going from the EventBridge Schedule to the Lambda?

Reply:
Good question - the `correlation_id` is not part of events but passed separately to the backend in the api call (see `_create_scheduled_job_and_set_started`).

Typically the event is created first, then it is sent into the queue for processing - this is when the `correlation_id` is created. Since we are doing the opposite (creating an event from the queue) I chose to pass the `correlation_id` separately to keep the event consistent with the other events.